.changeset/orange-singers-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-api": patch
+---
+
+Require the version of `@aws-sdk/client-s3` to be between 3.521.0 and 3.645.0
```

Comment:
This is in theory a breaking change. Wouldn't it be better if we moved the dependency from peer to normal? I believe this shouldn't break things for v6 projects

Reply:
> This is in theory a breaking change

I'm aware. I thought this was negligible since it just requires a minor up- or downgrade and probably won't affect most projects.

But making it a normal dependency is the better way to go